components/reusable-components/footer/Footer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Link from 'next/link';
+import { FaLinkedin, FaFacebook, FaInstagram, FaYoutube } from 'react-icons/fa';
+import styles from './footer.module.scss';
+
+const Footer = () => {
+  return (
+    <footer className={`${styles.footer}`}>
+      <div className={` display-s ${styles.footerBrand}`}>
+        <p>LearnHub.mk</p>
+      </div>
+
+      <div className={` title-l ${styles.contactSection}`}>
+        <p>CONTACT</p>
+        <p>contact@learnhub.mk</p>
```

Comment:
@Mikivitkovski here you can try and use `mailto: email address` for a href attribute (use an `<a>` tag) instead of a `<p>` tag